app/controllers/mixins/actions/vm_actions/right_size.rb

```diff
@@ -6,28 +6,26 @@ def vm_right_size
           assert_privileges(params[:pressed])
           # check to see if coming from show_list or drilled into vms from another CI
           rec_cls = "vm"
-          recs = params[:display] ? find_checked_ids_with_rbac(VmOrTemplate) : [find_id_with_rbac(VmOrTemplate, params[:id]).to_i]
-          if recs.length < 1
-            add_flash(_("One or more %{model} must be selected to Right-Size Recommendations") %
+          record = find_records_with_rbac(VmOrTemplate, checked_or_params).first
+          if record.nil?
+            add_flash(_("One %{model} must be selected to Right-Size Recommendations") %
```

Comment:
@PanSpagetka i think message should be "At least one %{model} must be selected....." or "One or more %{model} must be selected ...." what do you think?